### DISCUSSION TITLE
Loading Lightning model in PyTorch

### DISCUSSION THREAD

**Discussion by rahulvigneswaran** [Lightning-AI/pytorch-lightning#12041] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
How to load a model saved in PyTorch Lightning in Vanilla PyTorch?

**Comment by rohitgr7** [comment, marked as answer]
check this out: https://pytorch-lightning.readthedocs.io/en/latest/common/production_inference.html#id1
